src/python/grpcio_observability/grpc_observability/observability_util.h

```diff
@@ -0,0 +1,115 @@
+// Copyright 2023 gRPC authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// TODO(xuanwn): clean up includes
+#include <stdint.h>
+
+#include <algorithm>
+#include <condition_variable>
+#include <mutex>
+#include <queue>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "absl/strings/string_view.h"
+
+#include <grpc/status.h>
+
+#include "src/python/grpcio_observability/grpc_observability/constants.h"
+#include "src/python/grpcio_observability/grpc_observability/python_census_context.h"
+
+#ifndef OBSERVABILITY_MAIN_H
```

Comment:
Include guards need to go before any actual content. After the copyright notice, but before includes. Here and in all header files.